receiver/couchdbreceiver/scraper.go

```diff
@@ -92,10 +79,5 @@ func (c *couchdbScraper) appendMetrics(stats map[string]interface{}, rms pdata.R
 	c.recordCouchdbFileDescriptorOpenDataPoint(now, stats, errors)
 	c.recordCouchdbDatabaseOperationsDataPoint(now, stats, errors)
 
-	c.mb.Emit(md.ResourceMetrics().At(0).InstrumentationLibraryMetrics().At(0).Metrics())
-	if md.ResourceMetrics().At(0).InstrumentationLibraryMetrics().At(0).Metrics().Len() > 0 {
-		md.ResourceMetrics().At(0).CopyTo(rms.AppendEmpty())
-	}
-
-	return errors.Combine()
+	return c.mb.Emit(metadata.WithCouchdbNodeNameResourceAttribute(c.config.Endpoint)), errors.Combine()
```

Comment:
nit: The function names are always going to be very verbose if at a minimum we have `WithFooResourceAttribute`.  We could consider shortening the function name. I think it still reads fine to the scraper author. 
```suggestion
	return c.mb.Emit(metadata.WithCouchdbNodeName(c.config.Endpoint)), errors.Combine()
```

Reply:
I agree. Updated the code to the short version